kernel/kevinfs.h

```diff
@@ -44,6 +45,11 @@ struct kevinfs_inode {
 
 	uint32_t direct_addresses[FS_INODE_MAXBLOCKS];
 	uint32_t direct_addresses_len;
```

Comment:
Drop this field in favor of computing the number of valid blocks from the file size.